### PR TITLE
fix doc for http auth test

### PR DESCRIPTION
**Pull request by msyyc**
supplement for https://github.com/Azure/cadl-ranch/pull/341

### CODE REVIEW

File: packages/cadl-ranch-specs/cadl-ranch-summary.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Expect error code 403 and error body:
 
 - Endpoint: `get /authentication/http/custom/valid`
 
-Expects header 'Authorization': 'SharedAccessKey valid-key'
```
Comment (timotheeguerin) [inline packages/cadl-ranch-specs/cadl-ranch-summary.md:40]: So is that correct, I didn’t see the original pr but aren’t headed supposed to be case insensitive and we should update the validation to relate that. Isn’t that Pascal case version the most common?

Comment (weidongxu-microsoft) [reply packages/cadl-ranch-specs/cadl-ranch-summary.md:40]: They are same. Header name is case insensitive.

Comment (timotheeguerin) [reply packages/cadl-ranch-specs/cadl-ranch-summary.md:40]: Yeah so aren’t we being too restrictive here. (Aren’t some generator sending using Authorization) Wouldn’t the real fix just to make the validateHeader check be case insensitive

Comment (msyyc) [reply packages/cadl-ranch-specs/cadl-ranch-summary.md:40]: Got it. Since the header name is case insensitive, it is not necessary to update the doc.

**Pull request closed without merging**